controller/src/main/java/io/pravega/controller/task/KeyValueTable/TableMetadataTasks.java

```diff
@@ -101,7 +101,7 @@ public void initializeStreamWriters(final EventStreamClientFactory clientFactory
         }
         this.eventHelper = new EventHelper(clientFactory.createEventWriter(streamName,
                 ControllerEventProcessors.CONTROLLER_EVENT_SERIALIZER,
-                EventWriterConfig.builder().retryAttempts(Integer.MAX_VALUE).build()), this.executor, this.eventExecutor, hostId,
+                EventWriterConfig.builder().enableConnectionPooling(true).retryAttempts(Integer.MAX_VALUE).build()), this.executor, this.eventExecutor, hostId,
```

Comment:
A similar change is warrented @ `io.pravega.controller.task.Stream.StreamTransactionMetadataTasks#initializeStreamWriters`
This spawns 4 more connections to the Segment Store.